pkg/reconciler/reconciler.go

```diff
@@ -609,6 +622,20 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (res ctrl.
 		}
 	}()
 
+	if r.pauseReconcileAnnotation != "" {
+		if v, ok := obj.GetAnnotations()[r.pauseReconcileAnnotation]; ok {
+			if v == "true" {
+				log.Info(fmt.Sprintf("Resource has '%s' annotation set to 'true', reconcile stopped", r.pauseReconcileAnnotation))
+				u.UpdateStatus(
+					updater.EnsureCondition(conditions.Paused(corev1.ConditionTrue, conditions.ReasonPauseReconcileAnnotationSet, "")))
```

Comment:
I think we should perhaps be setting the remaining condition types to `Unknown` like it's done [here](https://github.com/stackrox/helm-operator/pull/29/files#diff-fc2bc6354d4506ced4adfd45dda2737dc917c217e282041d9af5bce57e514a85R641-R646)?